### PR TITLE
Consolidate logic between sqlite and mysql

### PR DESCRIPTION
**Pull request by shifucun**
* CreateTable() func to create tables as a one time setup
* Branch some SQL syntax that are not compatible between sqlite and mysql
* Use sqlDataPath to represent raw csv data path. TODO: make this working for GCS path.

### CODE REVIEW

File: docs/developer_guide.md [hunk 1]
```diff
@@ -82,14 +82,37 @@ root of this repo.
...
-echo mixer_api_key=<YOUR API KEY>
+export mixer_api_key=<YOUR API KEY>
 go run cmd/main.go \
     --use_sqlite=true \
     --use_bigquery=false \
     --use_base_bigtable=false \
     --use_branch_bigtable=false \
     --use_maps_api=false \
-    --sqlite_path=$PWD/data \
+    --sql_data_path=$PWD/data \
```
Comment (keyurva) [inline docs/developer_guide.md:92]: Wondering if this should be called something like `csv_data_path`?

File: internal/util/util.go [hunk 2]
```diff
@@ -594,9 +597,23 @@ func SQLInParam(n int) string {
...
+	case *mysql.MySQLDriver:
+		result := ""
+		for i := 0; i < n-1; i++ {
+			result += "SELECT ? UNION ALL "
+		}
+		result += "SELECT ?"
+		return result
+	default:
+		// This should not happen.
+		log.Println("Invalid sql driver")
```
Comment (keyurva) [inline internal/util/util.go:614]: Throw an exception in this case?

**Review by keyurva — APPROVED** [review]

**Review by pradh — APPROVED** [review]
Nice!

File: internal/util/util.go (hunk 2 above)
Comment (shifucun) [reply internal/util/util.go]: Done

File: docs/developer_guide.md (hunk 1 above)
Comment (shifucun) [reply docs/developer_guide.md:92]: In sqlite case, this is also the folder that contains the db files. I think sql_xx conveys the message that this is a folder for sql data?

**Comment by shifucun** [comment]
Thank you both for review!

**Pull request merged**